### DISCUSSION TITLE
Notice from Emporia of excessive data use

### DISCUSSION THREAD

**Discussion by magico13** [jertel/vuegraf#54] [General]
Hey, author of PyEmVue here. I know you've built this on top of my library so I just wanted you to be aware of this in case it's related to your application/users. Emporia contacted me about excessive API calls that are having a real impact on their database performance, primarily users requesting large amounts of historical data very often (eg years of data every few seconds). I don't think my own applications are responsible for that traffic and I'm not sure if yours is at all either, just wanted to let you know so you can assess it yourself. https://github.com/magico13/PyEmVue/issues/19

Thanks!

**Comment by jertel** [comment]
Thanks for the heads-up. I'll respond to the original thread so we can sort this out in one place.
